legal-api/src/legal_api/resources/v2/business/business.py

```diff
@@ -80,7 +80,6 @@ def get_businesses(identifier: str):
     recent_filing_json = CoreFiling.get_most_recent_filing_json(business.id, None, jwt)
     if recent_filing_json:
         business_json['submitter'] = recent_filing_json['filing']['header']['submitter']
-        business_json['lastModified'] = recent_filing_json['filing']['header']['date']
```

Comment:
Not required to override. lastModified will be updated in entity-filer while processing a filing